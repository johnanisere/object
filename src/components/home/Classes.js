import React from "react"
import OptionLayout from "../SectionLayout"
import List from "../../blocks/FlexibleContainer"

const App = () => <OptionLayout title="CLASSES">
    <List list="true" />
</OptionLayout>

export default App