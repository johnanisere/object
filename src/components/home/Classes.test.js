
import React from "react";
import { shallow } from "enzyme";

import Classes from "./Classes"
import OptionLayout from "../SectionLayout"
import List from "../../blocks/FlexibleContainer"

it("Classes component exists", () => {
    const wrapper = shallow(<Classes />);
    expect(wrapper.exists()).toBe(true);
});

it("Classes component renders", () => {
    const wrapper = shallow(<Classes />);
    expect(wrapper).toMatchSnapshot();
});

it(`Classes should contain "OptionLayout",  "List", `, () => {
    const wrapper = shallow(<Classes />);

    expect(wrapper.find(List)).toHaveLength(1);
    expect(wrapper.find(OptionLayout)).toHaveLength(1);
});