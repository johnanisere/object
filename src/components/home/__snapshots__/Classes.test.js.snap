// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Classes component renders 1`] = `ShallowWrapper {}`;
